### DISCUSSION TITLE
How can I get all the repository a user participate in  by using v4 api, thanks a lot !

### DISCUSSION THREAD

**Discussion by YuaJuan** [community/community#24836] [Apps, API and Webhooks]
<p>I want to get all  the repository a user participate in. How should I do using v4 api ? Thanks.</p>

**Comment by parithiban** [comment, marked as answer]
<p><a class="mention" href="/yuajuan">@yuajuan</a> </p>
<p>Did you tried the below query</p>
<pre><code>{
	rateLimit {
		cost
		limit
		nodeCount
		remaining
		resetAt
	}
	viewer {
		repositories(first: 50) {
			nodes {
				name
				url
			}
			pageInfo {
				endCursor
				hasNextPage
			}
		}
	}
}
</code></pre>
